CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## [1.6.0] - 2023-02-28
+
+### Added
+
+- Credential Composer plugins are now supported (#3819, #3832, #3862, #3869)
+- K8s workload attestor now can emmit Sigstore selectors (#3159)
+- Scratch images now supports arm64 (#3607)
+- Scratch images are now signed using Sigstore (#3707)
+- `spire-server mint` CLI command now support the `-output` flag (#3800)
+- `spire-agent` CLI commands now support the `-output` flag (#3818)
+- Release images now add non-root users and default folders (#3811)
```

Comment:
Since we are promoting scratch images as official release images in this release, I think that this PR should be just part of a separate item, maybe something like "Scratch images are now the official release images"